tailwind.config.js

```diff
@@ -20,9 +20,10 @@ module.exports = {
       fontSize: {
         fb: '1.188rem',
       },
-    },
-    fontFamily: {
-      sans: ['Roboto', 'Segoe UI', 'Helvetica'],
+      fontFamily: {
+        // prettier-ignore
+        sans: ['Source Sans Pro', 'sans-serif'],
```

Comment:
add the remaining list of fonts in sans from this page: https://tailwindcss.com/docs/font-family
That will allow a fallback font if none of these fonts are present on a device